packages/benchmark-wasm/vitest.config.ts

```diff
@@ -5,4 +5,7 @@ const plugins = process.env.CODSPEED ? [codspeedPlugin()] : [];
 
 export default defineConfig({
 	plugins,
+	test: {
+		watch: false,
```

Comment:
This didn't really work in local anyway because vitest cannot tell that the wasm file changed. Maybe there's a way to make it work though (perhaps importing the wasm file directly somewhere so that it has it in its module graph)